1_fetch.R

```diff
@@ -320,5 +320,11 @@ p1_targets_list <- list(
                                                "TotDASqKM", "DivDASqKM", "SLOPE", 
                                                "LakeFract", "SurfArea")), 
              deployment = "main"
-  )
+  ),
+  
+  ##screening out NA and -9998, -9999 values
+  tar_target(p1_feature_vars_conus_screen,
+             p1_feature_vars_conus %>%
+               na.exclude() %>%
```

Comment:
Is na.exclude equivalent to the tidy drop_na() function? Drops rows with NA? If it is, then could use the tidy function format

Reply:
ok, have updated that.  Let me know if it is ok to merge and I will get onto the plots.

Reply:
You can merge. How many reaches are dropped from these filters?

Reply:
129974 (~4.8 percent).  Most of those are from the drop_na filter (112821 of them)